### DISCUSSION TITLE
Add hyperlink in formie E-Mail notification?

### DISCUSSION THREAD

**Discussion by mandrasch** [verbb/formie#1699] [Q&A] [answered]
Hi,

we have a form which is opened via https://example.com/my-form-page?entryID=23. 

We store the ID in a hidden formie field, it belongs to a craft entry.

We now want to send out a notification via E-Mail where a hyperlink is included.

Hyperlink should be: https://example.com/entry/[entryID] 

What's the easiest way to achieve this? A custom E-Mail template? https://verbb.io/craft-plugins/formie/docs/template-guides/email-templates 

I would imagine it like that

```twig
<!doctype html>
<html>
    <head>
        <title>Thanks for your email</title>
    </head>

    <body>
        <p> New request</p>
        
        <p>The request was submitted from: <a href="{{ submission.entryId }}">Entry</a></p>

        <p>Full submission:</p>
        {{ contentHtml }}
    </body>
</html>
```

Thanks for quick confirmation / hints! 

Cheers, 
Matthias

**Comment by engram-design** [comment, marked as answer]
Either a custom email template would do the job, or you could add Twig in the link for the notification content.

First, ensure that you can add a link to the rich text editor by [enabling the button](https://verbb.io/craft-plugins/formie/docs/get-started/configuration#rich-text-configuration). Add a link, enter the text that you like, and for the URL value, use shorthand Twig for the handle of the hidden field where you've stored the entryID. The value might be `{entryId}` if your field has a handle `entryId`.

**Reply by mandrasch** [reply]
Ah! 💡 Thanks very much! I saw the buttons, but didn't made the connection to the correct docs. Awesome!
